src/js/serial_backend.js

```diff
@@ -348,7 +348,7 @@ function onOpenVirtual() {
     processBoardInfo();
 
     update_dataflash_global();
-    sensor_status(FC.CONFIG.activeSensors);
+    sensor_status(FC.CONFIG.activeSensors, FC.GPS_DATA.fix);
```

Comment:
Check the value of FC.GPS_DATA.fix here, as the value has not been queried upon opening (or is 0 as in fc.js)

Reply:
right, the value dosn't matter here if undefined or 0, but I change to 0